### PR TITLE
Update ug user friendly

### PR DESCRIPTION
**Pull request by solomonng2001**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -70,7 +70,3 @@ shadowJar {
 }
 
 defaultTasks 'clean', 'test'
-
-run {
-    enableAssertions = true
-}
```
Comment (yorklim) [inline build.gradle:76]: Why is this removed? Did you forget to pull before starting on you edits?

**Review by yorklim — DISMISSED** [review]
LGTM. Just some minor edits.

File: build.gradle (hunk 1 above)
Comment (solomonng2001) [reply build.gradle:76]: Added it back

**Review by ReganChoy — COMMENTED** [review]
LGTM

Assertion has been re-enabled on the gradle file.

**Review by ReganChoy — APPROVED** [review]
